### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jimmywarting**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,3 +77,8 @@ known by unsupported browsers). There may be other API calls in filer.js that
 do not work, but I haven't tested them.
 
 [![Analytics](https://ga-beacon.appspot.com/UA-46812528-1/ebidel/idb.filesystem.js/README)](https://github.com/igrigorik/ga-beacon)
+
+Finall note
```
Comment (ebidel) [inline README.md]: Final

File: README.md [hunk 2]
```diff
@@ -77,3 +77,8 @@ known by unsupported browsers). There may be other API calls in filer.js that
...
+-----------
+If you would like to see native support in Edge and Firefox, make your vote count by going to modern.ie and bugzilla and vote for Filesystem API [here](https://bugzilla.mozilla.org/show_bug.cgi?id=910387) and [here](https://dev.modern.ie/platform/status/filewriter/).
+Don't post anoying comments like +1 or "we want this NOW", that is what the voting system is for ;)
```
Comment (ebidel) [inline README.md]: I don't think you need this last sentence :)

**Comment by ebidel** [comment]
BTW, I don't think there's interest in this version of the FS API. The newer proposal is different.

**Comment by jimmywarting** [comment]
Just of curiosity would you want (to) help creating a new filesystem api based on this new filesystem that is filled with promises, Writable & Readable Stream and potentially everything else that is good about it.

Would be awesome to have some async/await kind of api where you also can pipe streams using web-stream-polyfill
Have a little hard time wrapping my head around the new spec.

**Comment by ebidel** [comment]
I'm not sure any browsers support it.

**Comment by jimmywarting** [comment]
Mmm, no 😆 

Would be hard to just follow a half spec'd api that is currently [drafted](http://w3c.github.io/filesystem-api/)

If I would write a such polyfill then i would do it with babel and 1 or 2 polyfill maybe

**Pull request closed without merging**